client/packages/requisitions/src/ResponseRequisition/DetailView/ResponseLineEdit/ResponseLineEdit.tsx

```diff
@@ -32,52 +31,33 @@ export const ResponseLineEdit = ({
     useDraftRequisitionLine(currentLine);
   const { next, hasNext } = useNextResponseLine(currentLine);
   const { data } = useResponse.line.stats(draft?.id);
-  const {
-    numberOfPacksToTotalQuantity,
-    numberOfPacksFromQuantity,
-    variantsControl,
-  } = usePackVariant(draft.itemId, draft.item.unitName ?? null);
 
   const height = useKeyboardHeightAdjustment(600);
 
   const tabs = [
     {
       Component: (
         <ResponseStoreStats
-          stockOnHand={numberOfPacksFromQuantity(
-            data?.responseStoreStats.stockOnHand || 0
-          )}
-          incomingStock={numberOfPacksFromQuantity(
-            data?.responseStoreStats.incomingStock || 0
-          )}
-          stockOnOrder={numberOfPacksFromQuantity(
-            data?.responseStoreStats.stockOnOrder || 0
-          )}
-          requestedQuantity={numberOfPacksFromQuantity(
-            data?.responseStoreStats.requestedQuantity || 0
-          )}
-          otherRequestedQuantity={numberOfPacksFromQuantity(
+          stockOnHand={data?.responseStoreStats.stockOnHand || 0}
```

Comment:
Do we want to or need to retain this ability to display SOH or other variables in something other than unit quantity?

Reply:
Don't think so... maybe we will one day, but I think better to YAGNI for now and implement as and how we need it, if we need it down the line :grin: